site.yml

```diff
@@ -45,6 +45,14 @@
       tags: ['infra', 'rabbitmq']
   environment: "{{ env_vars|default({}) }}"
 
+- name: setup serverspec
+  hosts: all
+  roles:
+    - role: serverspec
+      tags: ['serverspec']
+      when: serverspec.enabled|bool
+  environment: "{{ env_vars|default({}) }}"
```

Comment:
To be clear, I'm okay with it being its own role, to match SC and other repos, but I'm less okay with a unexpressed dependency.